### DISCUSSION TITLE
Is it possible to have a Select inside a text input? for a phone input with flag selection

### DISCUSSION THREAD

**Discussion by escobar5** [mantinedev/mantine#1740] [Q&A]
Hi,

I'm wondering if it is possible to do something like this with mantine:

![img](https://twilio-cms-prod.s3.amazonaws.com/images/E4WOMTrtKny5eD0FqJCnEKmcbM5iAKxSa3ZT2-ALYIrzrL.width-500.png)

**Comment by rtivital** [comment]
You can use Select with `icon` and `rightSection` props, example – https://ui.mantine.dev/component/currency-input

**Reply by escobar5** [reply]
It must be on the right though, right?

**Reply by rtivital** [reply]
It can be on the left side if you use `icon` instead of `rightSection`

**Reply by escobar5** [reply]
If I put a select in the `icon` property the layout breaks. But I've managed to do it with two separate controls and just a little css, thank you very much
